packages/admin/src/routes/entities/modules/path/sagas.js

```diff
@@ -120,7 +133,14 @@ export function* loadCurrentViewInfo({payload: {location}}) {
         currentViewInfo.key = path.key
         currentViewInfo.level = 1
 
-        yield call(addRecordToBreadcrumbs, path.entity + '/' + path.key, path.entity, path.key, breadcrumbs)
+        const bc = yield call(
+          addRecordToBreadcrumbs, path.entity + '/' + path.key, path.entity, path.key, breadcrumbs
+        )
+
+        if (bc && bc.error) {
+          yield put(actions.setCurrentViewInfo(pathname, {...currentViewInfo, error: bc.error}))
+          return
+        }
```

Comment:
this method starts looking dangerous... or continues.. 🙃 💀

Reply:
yes thats why i created a task to refactor and add unit tests. see TOCDEV-2326